jkube-kit/jkube-kit-smallrye/src/test/java/org/eclipse/jkube/smallrye/enricher/SmallRyeHealthCheckEnricherTest.java

```diff
@@ -20,33 +20,36 @@
 import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
 import org.eclipse.jkube.kit.common.Dependency;
 import org.eclipse.jkube.kit.common.JavaProject;
+import org.eclipse.jkube.kit.common.KitLogger;
 import org.eclipse.jkube.kit.config.resource.PlatformMode;
 import org.eclipse.jkube.kit.config.resource.ProcessorConfig;
 import org.eclipse.jkube.kit.enricher.api.JKubeEnricherContext;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.io.ByteArrayOutputStream;
 import java.io.File;
-import java.util.Collections;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Properties;
 import java.util.function.Supplier;
 
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
-import static org.mockito.Answers.RETURNS_DEEP_STUBS;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
 class SmallRyeHealthCheckEnricherTest {
   private JKubeEnricherContext context;
   private JavaProject javaProject;
   private Properties properties;
   private KubernetesListBuilder klb;
+  private ByteArrayOutputStream out;
 
   @BeforeEach
   void setup() {
     properties = new Properties();
     ProcessorConfig processorConfig = new ProcessorConfig();
```

Comment:
Since you're not using it anymore, Could you please remove it?

Reply:
ok sure, @rohanKanojia  I will do that and after that I will create pull request